code/game/objects/items/devices/enginepicker.dm

```diff
@@ -14,8 +14,12 @@
 	var/list/list_enginebeacons = list()
 	var/isactive = FALSE
 
+/obj/item/enginepicker/Destroy()
+	list_enginebeacons.Cut()
+	return ..()
+
 /obj/item/enginepicker/attack_self(mob/living/carbon/user)
-	if(usr.stat || !usr.canmove || usr.restrained())
```

Comment:
@FreeStylaLT why. WHY?!

Reply:
lol, also shouldn't we use `user` instead of `usr` here since its passed in?

Reply:
yeah

Reply:
😅